tests/test_closing.py

```diff
@@ -105,11 +103,12 @@ def test_closing_while_disconnected(node_factory, bitcoind):
     l2.stop()
 
     # The close should still be triggered afterwards.
-    with pytest.raises(RpcError, match=r'Channel close negotiation not finished'):
-        l1.rpc.close(chan, False, 0)
+    fut = executor.submit(l1.rpc.close, chan, 0)
```

Comment:
`check-python` is failing this:

>      tests/test_closing.py:106:11: F405 'executor' may be undefined, or defined from star imports: fixtures

Looks like need to add `executor` to the arguments of `test_closing_while_disconnected`?